### PR TITLE
Added immersive reader cards to conversation sample

### PR DESCRIPTION
**Pull request by Prithvi-MSFT**

### CODE REVIEW

**Review by Pawan-MSFT — APPROVED** [review]
looks fine, approving it!!

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving the PR.

File: samples/bot-conversation/nodejs/resources/UserMentionCardTemplate.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
   "type": "AdaptiveCard",
-  "version": "1.0",
+  "version": "1.5",
   "body": [
```
Comment (ChetanSharma-msft) [inline samples/bot-conversation/nodejs/resources/UserMentionCardTemplate.json:5]: Speak property is missing here:
image

**Review by ChetanSharma-msft — CHANGES_REQUESTED** [review]
Added my review comment.

File: samples/bot-conversation/nodejs/resources/UserMentionCardTemplate.json (hunk 1 above)
Comment (Prithvi-MSFT) [reply samples/bot-conversation/nodejs/resources/UserMentionCardTemplate.json:6]: Added

File: samples/bot-conversation/nodejs/bots/teamsConversationBot.js [hunk 2]
```diff
@@ -33,6 +34,8 @@ class TeamsConversationBot extends TeamsActivityHandler {
                 await this.messageAllMembersAsync(context);
             } else if (text.includes('who')) {
                 await this.getSingleMember(context);
+            } else if (text.includes('immersivereader')) {
```
Comment (Wajeed-msft) [inline samples/bot-conversation/nodejs/bots/teamsConversationBot.js:37]: I believe we need to update the welcome message adaptive card to include this option. 
image

**Review by Wajeed-msft — APPROVED** [review]

File: samples/bot-conversation/nodejs/bots/teamsConversationBot.js (hunk 2 above)
Comment (Prithvi-MSFT) [reply samples/bot-conversation/nodejs/bots/teamsConversationBot.js:37]: Added

**Comment by Prithvi-MSFT** [comment]
Build Pipeline Status - All builds passed
image

**Review by ChetanSharma-msft — APPROVED** [review]
Changes looks fine, approving it.

**Pull request merged**